src/examples/props.scss

```diff
@@ -6,45 +8,74 @@ h2 {
 }
 
 table {
+    display: block;
     width: 100%;
     padding: 0;
     margin-bottom: 50px;
-    table-layout: fixed;
-    box-shadow: 0 0 0 1px #ced4de;
-    background-color: transparent;
+
     border-spacing: 0;
     border-collapse: collapse;
     border-style: hidden;
-    border-radius: 5px;
-    font-size: 16px;
-    color: #13161f;
+    border: 1px solid var(--contrast-400);
+    border-radius: 4px;
 
-    thead {
-        color: #7d899c;
-        background: #eef1f5;
-    }
+    color: var(--contrast-1400);
+    -webkit-font-smoothing: antialiased;
+    background-color: var(--contrast-200);
 
-    thead th {
-        text-align: left;
+    thead {
+        border-radius: 4px 4px 0 0;
+        border-bottom: 1px solid var(--contrast-400);
+        color: var(--contrast-900);
         font-weight: 400;
-        padding: 20px 20px;
-    }
+        background: var(--contrast-200);
+
+        th {
+            text-align: left;
+            padding: 4px 8px 4px 12px;
+            font-size: 14px;
+
+            &:first-child {
+                border-top-left-radius: 4px;
+            }
 
-    tbody td {
-        padding: 12px 20px;
-        line-height: 2;
-        font-weight: 200;
-        vertical-align: top;
+            &:last-child {
+                border-top-right-radius: 4px;
+            }
+        }
     }
 
-    tbody > tr {
-        display: table-row;
-        border-top: 1px solid #ced4de;
+    tbody {
+        td {
+            padding: 8px 12px 12px 12px;
+            vertical-align: top;
+            font-size: 14px;
+        }
+        tr {
+            display: table-row;
+            &:nth-child(even) {
+                background-color: var(--contrast-300);
```

Comment:
Looking very nice! But, something a little weird happens when I go back to the landing page after visiting one of the component pages:

![image](https://user-images.githubusercontent.com/154725/85851673-6c28e280-b7af-11ea-9f5f-f1f1bb2199a6.png)

The reason it only happens after going to a component page and back is probably that this stylesheet isn't loaded until a component page is visited. But why only some of the rules seem to apply I have no idea.

Switching the props-component to use shadow-dom solves it, so I'll add a small commit doing that.

Reply:
@adrianschmidt i think those tables for some reason are not using the same tables which are used in props, because they have this weird `fsncwJ` class name and their own CSS. Do you know where they come from?
![image](https://user-images.githubusercontent.com/35954987/85852521-0e959580-b7b1-11ea-86c1-8013d2b410d7.png)

Reply:
:zap:

Reply:
> Do you know where they come from?

Yeah, they are created from our markdown:

![image](https://user-images.githubusercontent.com/154725/85852855-c165f380-b7b1-11ea-862e-565ab25a2cc4.png)

The styling applied is whatever the default docz theme applies. So that probably had some selectors with higher specificity and that's why only some of the styling was applied to that table.

Reply:
Just don't spend *too* much time styling the docz-app. It's really nice to have it look better, but hopefully we'll get Johan's [kompendium](https://github.com/jgroth/kompendium) to a state where it can replace docz sometime within the foreseeable future 🙂

But stuff that is very generic, like these new styles for tables, we can just lift straight over to kompendium if we like to, so it's not wasted effort.

Reply:
Awesome! No I won't spend too much time. This one was a big pain to look at every time, and I just thought it's so generic for tables, so we can move it to Johan's solution as well.

I'll just so some quick fixes to markdown tables too, so they sync